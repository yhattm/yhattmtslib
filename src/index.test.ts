import * as yhattmlibts from './index';

describe('index', () => {
  it('index', async () => {
    expect(yhattmlibts).toMatchSnapshot();
  });
});
