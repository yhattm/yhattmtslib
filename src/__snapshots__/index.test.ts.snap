// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index index 1`] = `
Object {
  "gas": Object {
    "GetPrice": [Function],
  },
  "gpio": Object {
    "setHigh": [Function],
    "setHighToLow": [Function],
    "setLow": [Function],
  },
  "invest": Object {
    "GetExchanges": [Function],
    "GetGold": [Function],
    "GetOption": [Function],
    "GetStock": [Function],
  },
  "mqtt": Object {
    "connect": [Function],
  },
  "token": Object {
    "getAWS": [Function],
  },
}
`;
